### PR TITLE
Run all workers by default from CLI or rake. Resolves #110

### LINKED ISSUE #110: `sneakers work` Should run all workers in app/workers
Major nitpick, but it'd be nice if the base command:

`sneakers work`

Looked up workers in `app/workers` or `lib/workers` and ran the ones that are Sneakers workers. That way, when I add a worker, I don't have to remember to update that command.

### PR DESCRIPTION
**Pull request by stevenkaras**
This should make it much more convenient to run all workers declared in an app

### CODE REVIEW

**Comment by jondot** [comment]
Looks like a good solution - would you mind taking a look at the failures for the build? (you can ignore the Ruby 1.9.3 one, as we'll be moving that one out)

Thanks!

**Comment by stevenkaras** [comment]
Looks like the test runner isn't unloading the worker after each test case. I think I tested against 2.1 locally, but I'll give it a shot with 2.0 and 2.2, see if I can reproduce the issue.

**Comment by stevenkaras** [comment]
I managed to reproduce this, and also in Ruby 2.1 with test seed 37892. I'll need some help figuring out if it's just unloading the workers properly, or if there's something else wrong here.

**Comment by gabrieljoelc** [comment]
We could really use this. Any way we can move this along?

**Comment by gabrieljoelc** [comment]
thanks! Woot! Green check marks!

**Comment by gabrieljoelc** [comment]
@jondot you willing to merge this now?

**Comment by jondot** [comment]
Sure am. Hey @stevenkaras - happy to see we're on the same time zone :)
Beer on me, whenever you'd like :)

File: lib/sneakers/tasks.rb [hunk 1]
```diff
@@ -13,9 +13,13 @@
       ::Rails.application.eager_load!
     end
 
-    workers, missing_workers = Sneakers::Utils.parse_workers(ENV['WORKERS'])
+    if ENV["WORKERS"].nil?
+      workers = Sneakers::Workers::Classes
```
Comment (gabrieljoelc) [inline lib/sneakers/tasks.rb]: This should be `Sneakers::Worker::Classes` so it's causing things to break :(

Comment (stevenkaras) [inline lib/sneakers/tasks.rb]: :scream:

**Pull request merged**